File: logica-de-programacao/secao-13/vetores.py
n = int(input("Quantos valores voce vai digitar: "))

vet = [0 for x in range(n)]

for i in range(0, n):
    vet[i] = float(input("Digite um numero: "))

print("Vetor digitado: ")
for i in range(0, n):
    print(vet[i])

File: logica-de-programacao/secao-11/base/vetores.java
package base;

import java.util.*;

public class vetores {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int N, i;

        System.out.print("Quantos numeros voce vai digitar? ");
        N = sc.nextInt();

        double[] vet = new double[N];

        for (i = 0; i < N; i++) {
            System.out.print("Digite um numero: ");
            vet[i] = sc.nextDouble();
        }

        System.out.println("\nNUMEROS DIGITADOS:");
        
        for (i = 0; i < N; i++) {
            System.out.println(vet[i]);
        }

        sc.close(); 
    }
}
